packages/navigation/types/Navigation.js

```diff
@@ -55,25 +55,4 @@ export type RouteNames =
   | HomepageStackNavigatorRouteNames
   | PlaygroundNavigationRouteNames;
 
-export type NavigationListener = (
-  key: 'willBlur' | 'willFocus' | 'didFocus' | 'didBlur',
-  // See payload https://reactnavigation.org/docs/en/navigation-prop.html#addlistener-subscribe-to-updates-to-navigation-lifecycle
-  callback: (payload: Object) => void,
-) => { remove: () => void };
-
-/**
- * @see https://reactnavigation.org/docs/navigators/navigation-prop
- */
-export type Navigation = {
-  navigate: ({
-    routeName: RouteNames,
-    key: string, // should be unique
-    params?: NavigationStateParameters,
-  }) => void,
-  state: {
-    params: NavigationStateParameters,
-  },
-  setParams: (newParameters: NavigationStateParameters) => void,
-  goBack: (key?: string | null) => void,
-  addListener: NavigationListener,
-};
+export type Navigation = NavigationScreenProp<*>;
```

Comment:
Please do not use this `*` type. This existential type has been deprecated.

Reply:
Should be or `NavigationStateParameters` or dynamic type as I explained in the other comment.

Reply:
> Should be or...

I think Github messed up your comment here? Did you mean _"Should be `<something here>` or"_? 😊